ccmlib/repository.py

```diff
@@ -37,12 +36,19 @@ def setup(version, verbose=False):
     if version.startswith('git:'):
         clone_development(GIT_REPO, version, verbose=verbose)
         return (version_directory(version), None)
-
     elif version.startswith('local:'):
-        if LOCAL_GIT_REPO is None:
-            raise CCMError("LOCAL_GIT_REPO is not defined!")
-        clone_development(LOCAL_GIT_REPO, version, verbose=verbose)
-        return (version_directory(version), None)
+        # local: slugs take the form of: "local:/some/path/:somebranch"
+        try:
+            _, path, branch = version.split(':')
+        except ValueError:
+            raise CCMError("local version ({}) appears to be invalid. Please format as local:/some/path/:somebranch".format(version))
+
+        clone_development(path, version, verbose=verbose)
+        version_dir = version_directory(version)
+
+        if version_dir is None:
+            raise CCMError("Path provided in local slug appears invalid.")
```

Comment:
I'm fine with no insight, that sounds infeasible to implement, but including the path would be great?